profile/README.md

```diff
@@ -1,26 +1,32 @@
+## Stacks: Unleash the Bitcoin economy with the leading Bitcoin L2
 
-📚 Ready to learn more about building on Bitcoin via Stacks? [Read the docs](https://docs.stacks.co/docs/intro).
+🟧 [Stacks](https://www.stacks.co/) is a Bitcoin Layer 2 built for scalability, security, and ease of enabling Bitcoin capital. [Learn more](https://www.stacks.co/learn/introduction)
 
-🏛️ Want to learn more about governance, funding, and other support? [Visit the Stacks Foundation GitHub](https://github.com/stacksgov).
+📚 Learn more about building on Bitcoin via Stacks: [Primer](https://www.stacks.co/build/get-started), [Docs](https://docs.stacks.co/docs/intro).
 
-🛠️ Want to learn more about the developer tooling fueling the Bitcoin L2 ecosystem? Visit the [Hiro Systems GitHub](https://github.com/hirosystems).
+🆕 Find the latest news: [Hourly](https://twitter.com/stacks), [Daily](https://stacks.co/blog), [Weekly](https://stackssnacks.com/), [Monthly](https://newsletters.stacks.org/)
 
-🌆 Want to learn more about the companies building the decentralized economy on Bitcoin? [Visit the Trust Machines GitHub](https://github.com/Trust-Machines).
+🌱 Learn more about funding and support for your Stacks project: [Fuel your project](https://www.stacks.co/build/fuel-your-project).
 
-🫂 Want to learn about community-led contributions to the Stacks Bitcoin L2? [Visit the BitcoinL2 Labs GitHub](https://github.com/BitcoinL2-Labs).
+🤝 Browse infrastructure and integrations Stacks supports: [Browse the Ecosystem](https://www.stacks.co/explore/ecosystem?category=All+Teams#tools)
+
+📊 Find data covering the Stacks network: [Data Tools](https://www.stacks.co/explore/ecosystem?category=Data#tools), [Explorer](https://explorer.hiro.so/)
+
+📱 Learn more about the apps unlocking the Bitcoin economy: [Browse Apps](https://www.stacks.co/explore/ecosystem?category=All+Teams#apps).
+
+🫂 Learn about how work gets done in the Stacks ecosystem: [Working Groups](https://github.com/orgs/stacks-network/discussions/504), [Visit the Stacks Foundation](https://stacks.org/grants).
 
 👨‍🌾 Want to see how all these people (and more) work together as a [Team of Teams](https://www.mcchrystalgroup.com/insights/what-kind-of-leader-can-lead-a-team-of-teams-the-6-principles-of-leading-like-a-gardener/)? Visit the [Stacks Core Engineering](https://github.com/stacks-network/stacks/blob/master/sBTC/stacks_core_engineering.md).
 
 _**Note**: All Blockstack v1 repositories have been moved to a new github org [stacks-archive](https://github.com/stacks-archive)_
 
 
+<br>
 <p align="left">
-    <a href="https://stacks.co"><img alt="Stacks" src="https://github.com/stacks-network/.github/blob/master/profile/images/banner.png" width="80%" style="margin-top:25px"/></a>
+    <a href="https://stacks.co"><img alt="Stacks" src="https://github.com/stacks-network/.github/assets/6131188/635c8f40-fa26-4ea4-a6dd-e56cb3b1a178" width="80%" style="margin-top:25px"/></a>
```

Comment:
same as above comment - if we're not using the static images in the repo, we should remove them (or overwrite the current images)

Reply:
I don't really follow, I did overwrite the current images. The new ones are what I want there and they've been uploaded to Github like the previous images were. I confirmed in the preview that the updated logo and updated bottom banner were showing.